native/packages/react-native-bpk-component-card/src/BpkCard-test.android.js

```diff
@@ -21,11 +21,19 @@
 import commonTests from './BpkCard-test.common';
 
 jest.mock('react-native', () => {
+  const MARSHMALLOW = 23;
   const reactNative = jest.requireActual('react-native');
   jest
     .spyOn(reactNative.Platform, 'select')
     .mockImplementation(obj => obj.android || obj.default);
   reactNative.Platform.OS = 'android';
+
+  // We rely on BpkTouchableNativeFeedback to handle the ripple effect on older devices
+  // so in this test suite we're assuming API > 23 where the ripple is supported
+  Object.defineProperty(reactNative.Platform, 'Version', {
+    value: MARSHMALLOW,
```

Comment:
This is to prevent the warning
```
Requested foreground ripple, but it is not available on this version of Android. Consider calling TouchableNativeFeedback.canUseNativeForeground() and using a different Touchable if the result is false.
```
we rely on `BpkToucheableNativeFeedback` to provide the correct ripple effect, so it's safe to test API >= 23